iotdb/src/main/java/org/apache/iotdb/db/engine/filenode/FileNodeProcessor.java

```diff
@@ -263,6 +269,11 @@ public FileNodeProcessor(String fileNodeDirPath, String processorName)
       registStatMetadata();
       statMonitor.registStatistics(statStorageDeltaName, this);
     }
+    try {
+      versionController = new SimpleFileVersionController(fileNodeDirPath);
```

Comment:
How about create a VersionControllerFactory. 
Latter we can modify the factory to switch the implementation of the versionController, otherwith SysTimestampVersionController is meaningless..

Reply:
Not quite necessary, SysTimestampVersionController is for test.